recipe/meta.yaml

```diff
@@ -53,7 +53,7 @@ requirements:
     - twisted
     - autobahn
     - six
-    - protobuf
+    - protobuf 3.0.*
```

Comment:
actually it should be "protobuf 3.4.*", see https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L85

Reply:
Current package "/pvpython: error while loading shared libraries: libprotobuf.so.10: cannot open shared object file: No such file or directory". Maybe it works if we pin it to the newer version during compilation.